data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -60,75 +63,144 @@
+    field_names.POVERTY_LESS_THAN_100_FPL_FIELD
+    + field_names.PERCENTILE_FIELD_SUFFIX: "P100_PFS",
     field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-    + field_names.PERCENTILE_FIELD_SUFFIX,
-]
+    + field_names.PERCENTILE_FIELD_SUFFIX: "P200_PFS",
+    field_names.LEAD_PAINT_FIELD
+    + field_names.PERCENTILE_FIELD_SUFFIX: "LPF_PFS",
+    field_names.NPL_FIELD 
+    + field_names.PERCENTILE_FIELD_SUFFIX: "NPL_PFS",
+    field_names.RMP_FIELD 
+    + field_names.PERCENTILE_FIELD_SUFFIX: "RMP_PFS",
+    field_names.TSDF_FIELD 
+    + field_names.PERCENTILE_FIELD_SUFFIX: "TSDF_PFS",
+    field_names.TOTAL_POP_FIELD: "TPF",
+    field_names.TRAFFIC_FIELD + field_names.PERCENTILE_FIELD_SUFFIX: "TF_PFS",
+    field_names.UNEMPLOYMENT_FIELD
+    + field_names.PERCENTILE_FIELD_SUFFIX: "UF_PFS",
+    field_names.WASTEWATER_FIELD
+    + field_names.PERCENTILE_FIELD_SUFFIX: "WF_PFS",
+    field_names.L_WATER: "L_WTR",
+    field_names.L_WORKFORCE: "L_WKFC",
+    field_names.L_CLIMATE: "L_CLT",
+    field_names.L_ENERGY: "L_ENY",
+    field_names.L_TRANSPORTATION: "L_TRN",
+    field_names.L_HOUSING: "L_HSG",
+    field_names.L_POLLUTION: "L_PLN",
+    field_names.L_HEALTH: "L_HLTH",
+    field_names.SCORE_L_COMMUNITIES: "SL_C",
+    field_names.SCORE_L + field_names.PERCENTILE_FIELD_SUFFIX: "SL_PFS",
+    field_names.EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD: "EPLRLI",
+    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD: "EALRLI",
+    field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD: "EBLRLI",
+    field_names.PM25_EXPOSURE_LOW_INCOME_FIELD: "PM25LI",
+    field_names.ENERGY_BURDEN_LOW_INCOME_FIELD: "EBLI",
+    field_names.DIESEL_PARTICULATE_MATTER_LOW_INCOME_FIELD: "DPMLI",
+    field_names.TRAFFIC_PROXIMITY_LOW_INCOME_FIELD: "TPLI",
+    field_names.LEAD_PAINT_MEDIAN_HOUSE_VALUE_LOW_INCOME_FIELD: "LPMHVLI",
+    field_names.HOUSING_BURDEN_LOW_INCOME_FIELD: "HBLI",
+    field_names.RMP_LOW_INCOME_FIELD: "RMPLI",
+    field_names.SUPERFUND_LOW_INCOME_FIELD: "SFLI",
+    field_names.HAZARDOUS_WASTE_LOW_INCOME_FIELD: "HWLI",
+    field_names.WASTEWATER_DISCHARGE_LOW_INCOME_FIELD: "WDLI",
+    field_names.DIABETES_LOW_INCOME_FIELD: "DLI",
+    field_names.ASTHMA_LOW_INCOME_FIELD: "ALI",
+    field_names.HEART_DISEASE_LOW_INCOME_FIELD: "HDLI",
+    field_names.LOW_LIFE_EXPECTANCY_LOW_INCOME_FIELD: "LLELI",
+    field_names.LINGUISTIC_ISOLATION_LOW_HS_EDUCATION_FIELD: "LILHSE",
+    field_names.POVERTY_LOW_HS_EDUCATION_FIELD: "PLHSE",
+    field_names.LOW_MEDIAN_INCOME_LOW_HS_EDUCATION_FIELD: "LMILHSE",
+    field_names.UNEMPLOYMENT_LOW_HS_EDUCATION_FIELD: "ULHSE",
+    field_names.COMBINED_UNEMPLOYMENT_2010: "CU2010",
+    field_names.COMBINED_POVERTY_LESS_THAN_100_FPL_FIELD_2010: "CPL1002010",
+    field_names.CENSUS_DECENNIAL_UNEMPLOYMENT_FIELD_2009: "CDUF2009",
+    field_names.FPL_200_SERIES: "FPL200S"
+}
 
 # columns to round floats to 2 decimals
 TILES_SCORE_FLOAT_COLUMNS = [
```

Comment:
Since 90% or so of the score columns are floats, should we change this to actually just highlight the small number of string columns? (That can be a later refactor, obviously.)

Reply:
yeah thats a good idea. i vote we can fix this later ^.^

Reply:
makes sense! maybe leave a #TODO or something?